blogposts/2021/api-documentation-generated-for-all-your-code.md

```diff
@@ -0,0 +1,84 @@
+
+* Surface usage examples from open source code.
+* Providing a single tool that works across all code, using LSIF data to provide an experience tailored towards your languages of choice.
+* Giving an amazing navigation and search experience.
+
+## Try it (experimental)
+
+Today, it's still in its very early stages—and only a few months into development. But you can try it out on Go code today:
+
+
+- [Explore encoding/json code](https://sourcegraph.com/github.com/golang/go/-/docs/encoding/json)
+- [Explore Go net/http code](https://sourcegraph.com/github.com/golang/go/-/docs/net/http)
+- [Explore Sourcegraph backend code](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/docs/cmd/frontend/backend)
+
+Want to try it out on your own Go repositories? Tweet [@sourcegraph](https://twitter.com/sourcegraph) with your GitHub repository name and we'll set it up for you, or [follow our documentation]([https://docs.sourcegraph.com/code_intelligence/apidocs](https://docs.sourcegraph.com/code_intelligence/apidocs)).
+
+## Improving rapidly
+
+Here's a little highlight of the changes we've made recently.
+
+### On by default, for everyone
+
+The most important announcement here is that API docs is now an on-by-default experiment for everyone! You can begin trying it out on Sourcegraph.com as well as your own Sourcegraph instance by clicking on the API docs button on any repository page:
+
+![API Docs button](https://sourcegraphstatic.com/blog/api-docs/api-docs-button.png)
+
+
+Once you click it, you'll be greeted with information about how to get started in a brand new welcome prompt we’ve added:
+
+![Welcome prompt](https://sourcegraphstatic.com/blog/api-docs/welcome-prompt.png)
+
+### Better sub-page navigation
+
+Within a Go package, there are many sub-packages. We've improved the way we render documentation subpages so you can skip over useless directories and get right to the content (but we're still hard at work improving navigation in API docs!):
+
+![Sub-page navigation](https://sourcegraphstatic.com/blog/api-docs/subpage-navigation.gif)
+
+### Understands the types of symbols, and uses iconography to communicate
+
+Due to API docs being built on code intelligence data, it understands exactly what symbols are and has detailed information about them. Whether symbols are deprecated, whether they're a function, variable, interface, etc. and the UI now communicates this information using icons:
+
+![Symbols information](https://sourcegraphstatic.com/blog/api-docs/symbols.gif)
+
+### A deep understanding of code
+
+We're paying attention to the small details, and really tailoring API docs to each language. We want an experience that makes sense for Go first, then we'll expand to more languages and build an experience that makes sense for another language.
```

Comment:
I asked this in the G Doc, are we choosing Go first because we use Go to develop Sourcegraph? Someone unfamiliar with our stack might wonder why Go.

Reply:
```suggestion
We're paying attention to the small details, and really tailoring API docs to each language. We want an experience that makes sense for Go first, because here at Sourcegraph we use primarily Go and TypeScript and want to ensure we build an experience Go developers love, then we'll expand to more languages and build an experience that makes sense in the context of other languages - a tailored experience for each.
```